tests/snippets/dict.py

```diff
@@ -95,8 +95,15 @@
 x[2] = 2
 x[(5, 6)] = 5
 
-with assertRaises(KeyError):
-    x["not here"]
+assert_raises(KeyError,
+              lambda: x['not here'],
+              "'not here'")
```

Comment:
I am not sure the `msg` `'not here'` here is actually helpful. The output is:
```py
AssertionError: KeyError was not raised: 'not here'
```
Please consider removing the `msg` or making it more helpful.

Reply:
Thanks, I got the idea of the function wrong.